src/Date.php

```diff
@@ -97,7 +97,7 @@ public function __construct($time = 'now', $tz = null)
         $testNow = Chronos::getTestNow();
         if ($testNow === null || !static::isRelativeOnly($time)) {
             $time = $this->stripTime($time, $tz);
-            parent::__construct($time, new DateTimeZone('UTC'));
+            parent::__construct($time, new DateTimeZone(date_default_timezone_get()));
```

Comment:
Remove `new DateTimeZone` to optimize default time zone in DateTime constructor.

Reply:
Can't do that, I get errors from PHP.  `TypeError: DateTime::__construct() expects parameter 2 to be DateTimeZone, string given` and `TypeError: DateTimeImmutable::__construct() expects parameter 2 to be DateTimeZone, string given`

Reply:
You don't pass a 2nd parameter. It will use the default automatically.